core/edit/edit_command.py

```diff
@@ -1,41 +1,177 @@
-from talon import Module, actions
+from talon import Module, actions, settings
 
 from .edit_command_actions import EditAction, run_action_callback
 from .edit_command_modifiers import EditModifier, run_modifier_callback
 
-# In some cases there already is a "compound" talon action for a given action and modifier
+mod = Module()
+mod.setting(
+    "edit_command_word_selection_delay",
+    type=int,
+    default=75,
+    desc="Sleep required between word selections",
```

Comment:
Can you explain a bit more as to why this is necessary?

Reply:
Added comment above the setting.